src/Microsoft.DotNet.Tools.Compiler.Native/project.json

```diff
@@ -25,7 +25,8 @@
         },
       "Microsoft.DotNet.ILCompiler": "1.0.1-*",
       "Microsoft.DotNet.ObjectWriter": "1.0.2-*",
-      "Microsoft.DotNet.RyuJit": "1.0.0-*"
+      "Microsoft.DotNet.RyuJit": "1.0.0-*",
```

Comment:
@schellap - when we publish the next version of ILCompiler/RyuJIt/ObjectWriter, what would be their version? Would we need to update this file to pick them up?